etc/kayobe/environments/ci-aio/automated-setup.sh

```diff
@@ -2,39 +2,48 @@
 
 set -eux
 
-cat << EOF | sudo tee -a /etc/hosts
-10.205.3.187 pulp-server pulp-server.internal.sms-cloud
-EOF
-
-if sudo vgdisplay | grep -q lvm2; then
-   sudo pvresize $(sudo pvs --noheadings | head -n 1 | awk '{print $1}')
-   sudo lvextend -L 4G /dev/rootvg/lv_home -r || true
-   sudo lvextend -L 4G /dev/rootvg/lv_tmp -r || true
-fi
-
 BASE_PATH=~
 KAYOBE_BRANCH=stackhpc/yoga
 KAYOBE_CONFIG_BRANCH=stackhpc/yoga
+KAYOBE_AIO_LVM=true
 
 if [[ ! -f $BASE_PATH/vault-pw ]]; then
     echo "Vault password file not found at $BASE_PATH/vault-pw"
     exit 1
 fi
 
-if type dnf; then
-    sudo dnf -y install git
-else
-    sudo apt update
-    sudo apt -y install gcc git libffi-dev python3-dev python-is-python3
+if sudo vgdisplay | grep -q lvm2; then
+   sudo pvresize $(sudo pvs --noheadings | head -n 1 | awk '{print $1}')
+   sudo lvextend -L 4G /dev/rootvg/lv_home -r || true
+   sudo lvextend -L 4G /dev/rootvg/lv_tmp -r || true
+elif $KAYOBE_AIO_LVM; then
+   echo "This environment is only designed for LVM images. If possible, switch to an LVM image.
+   
+   While it is not recommended, the environment can be adapted by:
+   Removing the lvm.yml group vars file in the ci-aio environment
+   Unsetting controller_lvm_groups in controllers.yml in the ci-aio environment
+   setting KAYOBE_AIO_LVM to false in this script."
+   exit 1
```

Comment:
You could argue that having taken the action to set `KAYOBE_AIO_LVM` to `false`, it would be reasonable to modify the config to unset controller_lvm_groups as you did before. Otherwise you'd need to use a custom branch.